src/test/java/uk/gov/companieshouse/charges/data/service/ChargesServiceTest.java

```diff
@@ -315,11 +292,31 @@ void when_connection_issue_in_db_on_find_in_delete_then_throw_service_unavailabl
             chargesService.deleteCharge( "x-request-id", chargeId);
         }
         catch (ResponseStatusException statusException)  {
-            Assert.assertEquals(HttpStatus.SERVICE_UNAVAILABLE, statusException.getStatus());
+            assertEquals(HttpStatus.SERVICE_UNAVAILABLE, statusException.getStatus());
         }
 
     }
 
+    @Test
+    void when_charge_id_exist_ani_invoke_chs_kafka_api_un_successfully_invoked_then_delete_charge() {
```

Comment:
"and" instead of "ani" typo - feel free to disregard.

Reply:
Refactoring of the older unit tests could be done as part of a tech debt ticket.